config/charts/templates/kube-prometheus-stack.tpl

```diff
@@ -2,7 +2,34 @@ nameOverride: "prometheus-operator"
 fullnameOverride: "prometheus-operator"
 
 defaultRules:
-  create: false
+  create: true
+  rules:
+    alertmanager: true
+    etcd: true
+    general: true
+    k8s: true
+    kubeApiserver: true
+    kubeApiserverAvailability: true
+    kubeApiserverError: true
+    kubeApiserverSlos: true
+    kubelet: true
+    kubePrometheusGeneral: true
+    kubePrometheusNodeAlerting: true
+    kubePrometheusNodeRecording: true
+    kubernetesAbsent: true
+    kubernetesApps: true
+    kubernetesResources: false
```

Comment:
because of an open issue here: https://github.com/prometheus-community/helm-charts/issues/471